src/main/webapp/js/adminAccountManagement.js

```diff
@@ -15,7 +15,7 @@ $(document).ready(function() {
     showFirstPage();
     updateEntriesCount();
     bindDeleteAccountAction();
-    AdminCommon.bindBackToTopButtons();
+    AdminCommon.bindBackToTopButtons('.back-to-top-left, .back-to-top-right');
```

Comment:
Is this in every admin page? If yes why not move the responsibility to AdminCommon itself rather than having everyone invoke it as and when they like (they'll probably forget)